lib/rspec/its/version.rb

```diff
@@ -1,5 +1,5 @@
 module RSpec
   module Its
-    VERSION = "1.2.0"
+    VERSION = "1.3.0"
```

Comment:
Can you drop this? We will bump this on release.

Reply:
yup, no problem